src/main/java/org/pac4j/vertx/handler/impl/CallbackHandler.java

```diff
@@ -51,75 +43,37 @@
     private final HttpActionAdapter httpActionHandler = new DefaultHttpActionAdapter();
     private final Vertx vertx;
     private final Config config;
+    private final boolean multiProfile;
 
     protected String defaultUrl = Pac4jConstants.DEFAULT_URL_VALUE;
```

Comment:
Vert.x conventions are where we extend these things we use options classes (see all over the place in the vert.x codebase). This is especially useful where we have a mixture of mandatory and optional settings which default. As far as possible for vert.x usage I will remain consistent with the established vert.x idioms.